docs/how-does-version-resolution-work-in-maven-and-gradle.md

```diff
@@ -0,0 +1,117 @@
+# How does version resolution work in Maven and Gradle?
+
+The build system that you choose for building your Java code determines which
```

Comment:
Not sure what to do about this, but there are other build systems that do something completely different. E.g Eclipse and Ant don't necessarily use the Maven repository system at all. SBT and Ant+Ivy do. This article is really about "Version resolution in the Maven Respository System" or some such.

Reply:
Our entire set of best practices ignores the Eclipse/Ant world, right? The practices are all in terms of the Maven repository system. Anyway, I will add a note that Eclipse/Ant are out of scope.

Reply:
no, not completely. JLBP-17, 18, 19, and 20 for instance apply equally well to ant. So does JLBP-1 and probably others.

Reply:
They *can* apply incidentally, but our focus is on build systems using the Maven repository system. 

If you want to increase our scope to Eclipse/Ant, we'd probably want to clarify throughout our best practices what the scope of each one is. For example, I have no idea if BOMs are useful (or possible) for Eclipse/ant.

Reply:
I have added a note about this.